File: crates/aft/tests/fixtures/inline_symbol/sample.ts
const BASE_URL = "https://api.example.com";

function add(a: number, b: number): number {
  const sum = a + b;
  return sum;
}

function main() {
  const x = 10;
  const y = 20;
  const result = add(x, y);
  console.log(result);
}

const double = (n: number): number => n * 2;

function caller() {
  const val = double(5);
  console.log(val);
}

File: crates/aft/tests/fixtures/inline_symbol/sample.py
BASE_URL = "https://api.example.com"

def add(a, b):
    total = a + b
    return total

def main():
    x = 10
    y = 20
    result = add(x, y)
    print(result)

def double(n):
    return n * 2

def caller():
    val = double(5)
    print(val)
